internal/config/config.go

```diff
@@ -84,68 +81,5 @@ func newEnvironment() environment {
 	vars.IntVar(&env.SLAHoursToClose, "hellper_sla_hours_to_close", 168, "SLA hours to close")
 
 	vars.Parse()
-	env.Messages = newMessages(env.Language)
```

Comment:
No package is using this field **Messages**?

Reply:
No, it isn't used at this time in any package.

Reply:
So you can delete this field also.